resources/views/partials/navbar.blade.php

```diff
@@ -1,40 +1,60 @@
-              </li>
-              <li class="nav-item mr-6 sart" role="presentation">
-                    <div class="dropdown">
-                        <a class="nav-link dropdown-toggle" id="report-dropdown" data-toggle="dropdown" href="#">Expense</a>
-                        <ul class="dropdown-menu" aria-labelledby="report-dropdown">
-                            <li class="nav-item">
-                                <a href="{{ route('expense.reports') }}" class="nav-link">Expenditure Reports</a>
-                            </li>
-                            <li class="nav-item">
-                                <a href="{{ route('expense.ministry') }}" class="nav-link">Ministry Expense</a>
-                            </li>
-                        </ul>
-                    </div>
-                </li>
-                <li class="nav-item mr-6 sart" role="presentation">
-                    <a class="nav-link" href="{{ route('ministries') }}">Ministries</a>
-                </li>
-                <li class="nav-item mr-6 sart" role="presentation">
-                    <a class="nav-link" href="{{ route('contractors') }}">Contractors</a>
-                </li>
-                <li class="nav-item mr-6 sart" role="presentation">
-                    <a class="nav-link" href="{{ route('about') }}">About us</a>
-                </li>
-                  <a class="nav-link" href=""><i class="fa fa-search inp"></i></a>
-          </ul>
-      </div>
-    </div>
-  </nav>
+        <div class="container-fluid outer-margin">
+            <a class="navbar-brand" href="/"><img src="{{asset('images/Logo.svg')}}"></a>
+            <!--<a class="navbar-brand" href="/"><img src="logo.png"></a>-->
+            <button data-toggle="collapse" class="navbar-toggler" data-target="#navcol-1">
+                <span><i class="fa fa-bars" aria-hidden="true"></i></span>
+            </button>
+          <div class="collapse navbar-collapse" id="navcol-1">
+              <ul class="nav navbar-nav ml-auto">
+                  <li class="nav-item" role="presentation" id="home">
+                      <a class="nav-link" href="{{ route('home') }}">Home</a>
+                  </li>
+                  <li class="nav-item" role="presentation" id="spending">
+                      <div class="dropdown">
+                        <a class="nav-link" href="#" role="button" id="dropdownMenuLink" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                          Spending
+                        </a>
+
+                        <div class="dropdown-menu dropdown-menu-spending" aria-labelledby="dropdownMenuLink">
+                          <a class="dropdown-item for-spending" href="{{ route('expense.reports') }}">Expenditure Report</a>
+                          <a class="dropdown-item for-spending" href="{{ route('expense.ministry') }}">Ministry Spending</a>
+                        </div>
+                      </div>
```

Comment:
Also, the font-size for the navbar is too big.

The banner on home page is too full and still needs some adjustments to fit the design.

Reply:
I'll look into it